business_requirement_deliverable/views/business_view.xml

```diff
@@ -0,0 +1,224 @@
+            </field>
+        </field>
+    </record>
+
+    <record id="view_business_requirement_tree" model="ir.ui.view">
+        <field name="name">business.requirement.tree</field>
+        <field name="model">business.requirement</field>
+        <field name="inherit_id" ref="business_requirement.view_business_requirement_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr='//tree/field[@name="priority"]' position='after'>
+                <field name="currency_id" invisible="1" groups="business_requirement_deliverable.group_business_requirement_estimation"/>
+                <field name="total_revenue"
+                       sum="Total revenue"
+                       widget='monetary' options="{'currency_field': 'currency_id'}"
+                       groups="business_requirement_deliverable.group_business_requirement_estimation"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="view_business_requirement_deliverable_tree" model="ir.ui.view">
+        <field name="name">business.requirement.deliverable.tree</field>
+        <field name="model">business.requirement.deliverable</field>
+        <field name="arch" type="xml">
+            <tree string="Business Requirement Deliverable" default_order="sequence">
+                <field name="sequence" widget="handle"/>
+                <field name="business_requirement_id"/>
+                <field name="name"/>
+                <field name="product_id"/>
+                <field name="qty"/>
+                <field name="uom_id"/>
+                <field name="sale_price_unit" groups="business_requirement_deliverable.group_business_requirement_estimation"/>
+                <field name="price_total" sum="Total price" groups="business_requirement_deliverable.group_business_requirement_estimation"/>
+                <field name="currency_id" groups="business_requirement_deliverable.group_business_requirement_estimation"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="view_business_requirement_deliverable_form" model="ir.ui.view">
+        <field name="name">business.requirement.deliverable.form</field>
+        <field name="model">business.requirement.deliverable</field>
+        <field name="type">form</field>
+        <field name="priority">20</field>
+        <field name="arch" type="xml">
+            <form string="Business Requirement Task Type">
+                <sheet>
+                    <div class="oe_button_box" name="button_box" groups="base.group_user">
+                         <button class="oe_inline oe_stat_button" name="portal_publish_button" type="object" icon="fa-globe">
+                             <label for="portal_published" />
+                             <field name="portal_published" widget="toggle_button"/>
+                         </button>
+                    </div>
+                    <group>
```

Comment:
The title is going to better hanging from the `<sheet>`

Reply:
Not sure how to do that, could you add suggested change?

Reply:
```suggestion
```

Reply:
(it's not possible to add multiline suggestions, so you'll have to fix indent though)

Reply:
get it it's about the indent